tests/models/openoil/test_openoil_custom_type.py

```diff
@@ -1,11 +1,15 @@
 import json
-from importlib import resources
+from importlib_resources import files
+
 from opendrift.models.openoil import OpenOil
 
+oiljson = files('opendrift.models.openoil.adios.extra_oils').joinpath('AD04011.json')
+
 def test_set_oil_type_json():
     o = OpenOil(loglevel=50)
 
-    d = json.loads(resources.read_text('opendrift.models.openoil.adios.extra_oils', 'AD04011.json'))
+    with open(oiljson) as f:
```

Comment:
@gauteh This change removes warnings related to deprecated importlib resources. Is this new solution ok?